lib/auth/getKerberos.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const requireOptional = require('require_optional');
+
+let Kerberos = null;
+let MongoAuthProcess = null;
+
+try {
+  const kerberos = requireOptional('kerberos');
+  if (kerberos) {
+    Kerberos = kerberos.Kerberos;
+    MongoAuthProcess = kerberos.processes.MongoAuthProcess;
+  }
+} catch (err) {
+  console.warn(err.message);
```

Comment:
I think the reason this was ignored in the first place is because it is meant to be optional. Rather, I think the warning should be suppressed if the module isn't found, but it should _rethrow_ for any other error